lib/cib/cib_ops.c

```diff
@@ -538,6 +538,34 @@ add_cib_object(xmlNode * parent, xmlNode * new_obj)
     return update_cib_object(parent, new_obj);
 }
 
+/* remove this function */
```

Comment:
lol, let's just drop the comment. I vaguely remember looking a little bit into this before. I don't remember exactly but I think the intent was that it would become unnecessary with improved error handling. Not worth any effort now ...